### PR TITLE
Development

### PR DESCRIPTION
**Pull request by rmauritsson**

### CODE REVIEW

**Comment by stratospherique** [comment]
### Pull Request under the review by Ahmed Mahfoudh TSE

File: README.md [hunk 1]
```diff
@@ -1,2 +1,18 @@
...
+
+The framework was used in cloning www.7diamonds.com website in this project. 
+
+#Collaborators
+1. Leonard Rwai Kanyesigye @rmauritsson
+2. Kingsley McSimon Ogbonna @KingsleyMcSimon
+
+#Link to Original Website Cloned
+https://7diamonds.com/
+
```
Comment (stratospherique) [inline README.md]: - You can add live preview link of both of your pages. use [githack](https://raw.githack.com/)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+                <img src="images/logo.png" alt="logo">
+            </div>
+            <div class="cart gd-ld-4">
+                <img src="https://image.flaticon.com/icons/png/512/131/131049.png" alt="cart">
+            </div>
+        </div>
+    </header>
+
+    <section class="main">
+
```
Comment (stratospherique) [inline index.html]: This code is responsible the big image. It looks fine but not centered. And it seems that it is causing a horizontal scroll on you page.

**Comment by stratospherique** [comment]
### That's it guys :clinking_glasses: :clinking_glasses: . Finish implementing these requirements and kindly re-submit a code review request :muscle: :muscle: .

File: README.md (hunk 1 above)
Comment (rmauritsson) [reply README.md]: Hello @stratospherique 
I have comments to make on your proposed changes.
1. The class that represents a row exists in our framework and it's called wrapper
2. We have classes that define column sizes as shown in the file `framework.html`
3. The classes have breakpoints forexample `
<div class="gd-ld-12 gd-md-12 gd-sd-12 gd-gt mainslider2" >
                    <div class="wrapper-items">
                        <h2>SUMMER SHIRTS</h2>
                        <button class="secondaryBtwn shop-btn">Shop Short</button>
                    </div>
   </div>`

File: index.html (hunk 2 above)
Comment (rmauritsson) [reply index.html]: Removed the horizontal scroll as well

File: README.md (hunk 1 above)
Comment (stratospherique) [reply README.md]: yes you have :clap: :clap: . But you can add more graduation and make them 12. Apart from that, if you have fixed the other issues you can submit another code review request. keep up the good work :muscle: :muscle:

**Comment by rmauritsson** [comment]
Thank You

**Pull request merged**